### PR TITLE
[RSS-23] Edit Profile Page

### PR DESCRIPTION
**Pull request by ObjectJosh**

### CODE REVIEW

File: src/components/Main/Main.jsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import LoginPage from '../LoginPage/LoginPage';
+import EditProfilePage from '../EditProfilePage/EditProfilePage';
 
 export default () => (
   <div className="App">
     <Switch>
       <Route exact path="/" render={() => <LoginPage />} />
+      <Route exact path="/test" render={() => <EditProfilePage />} />
```
Comment (jackfales) [inline src/components/Main/Main.jsx]:
```suggestion
```

File: src/components/Main/Main.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import LoginPage from '../LoginPage/LoginPage';
+import EditProfilePage from '../EditProfilePage/EditProfilePage';
```
Comment (jackfales) [inline src/components/Main/Main.jsx]:
```suggestion
```

File: src/components/EditProfilePage/EditProfilePage.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  };
+  const raiseError = () => {
+    if (name === '' || email === '' || phone === '') {
+      setError(true);
+      alert('Error: Field(s) empty');
+    }
+  };
+  const handleSave = () => {
+    console.log(email);
+    console.log(phone);
```
Comment (jackfales) [inline src/components/EditProfilePage/EditProfilePage.jsx]:
```suggestion
```

File: src/components/EditProfilePage/EditProfilePage.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    setName(e.target.value);
+    console.log(name);
+  };
+  const updateEmailField = (e) => {
+    setEmail(e.target.value);
+    console.log(email);
+  };
+  const updatePhoneField = (e) => {
+    setPhone(e.target.value);
+    console.log(phone);
```
Comment (jackfales) [inline src/components/EditProfilePage/EditProfilePage.jsx]:
```suggestion
```

File: src/components/EditProfilePage/EditProfilePage.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  const [email, setEmail] = useState('');
+  const [phone, setPhone] = useState('');
+  const [error, setError] = useState(false);
+  const updateNameField = (e) => {
+    setName(e.target.value);
+    console.log(name);
+  };
+  const updateEmailField = (e) => {
+    setEmail(e.target.value);
+    console.log(email);
```
Comment (jackfales) [inline src/components/EditProfilePage/EditProfilePage.jsx]:
```suggestion
```

File: src/components/EditProfilePage/EditProfilePage.jsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  border: none;
+`;
+export default () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [phone, setPhone] = useState('');
+  const [error, setError] = useState(false);
+  const updateNameField = (e) => {
+    setName(e.target.value);
+    console.log(name);
```
Comment (jackfales) [inline src/components/EditProfilePage/EditProfilePage.jsx]:
```suggestion
```

**Review by jackfales — COMMENTED** [review]
Look good just remove the lines I commented on

**Review by ellabuysse — APPROVED** [review]
Looks good!

**Pull request merged**